solution-base/build.sh

```diff
@@ -194,7 +199,11 @@ function render_mongodb_sharded_yamls()
         --set 'configsvr.persistence.selector.matchLabels.app\.kubernetes\.io/name=mongodb-sharded-config' \
         --set 'configsvr.persistence.selector.matchLabels.app\.kubernetes\.io/part-of=zenko' \
         --set "configsvr.mongodbExtraFlags={--setParameter rollbackTimeLimitSecs=259200}" \
-        --set "shardsvr.dataNode.mongodbExtraFlags={--setParameter rollbackTimeLimitSecs=259200}" \
+        --set mongos.resources.requests.memory=${MONGODB_MONGOS_RAM_REQUEST} \
+        --set mongos.resources.limits.memory=${MONGODB_MONGOS_RAM_LIMIT} \
+        --set "shardsvr.dataNode.mongodbExtraFlags={--wiredTigerCacheSizeGB=${MONGODB_WIREDTIGER_CACHE_SIZE_GB}}" \
+        --set shardsvr.dataNode.resources.limits.memory=${MONGODB_SHARDSERVER_RAM_SAFE_LIMIT} \
```

Comment:
why "_SAFE_" ?

there is nothing safe here (or even explicit relationship with `MONGODB_WIREDTIGER_CACHE_SIZE_GB` in zenko), it is really just the k8s memory limit : so best keep if consistent with the other variables...

Reply:
The safe means that the k8s limit is not formally required as long as the wiredtiger cache is limited, it only protects against memory leak (we never seen that with mongod yet).
I will remove it and make the name clearer